staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -88,6 +88,14 @@ const (
 	// Add support for distributed tracing in the API Server
 	APIServerTracing featuregate.Feature = "APIServerTracing"
 
+	// owner: @seans3
+	// kep: http://kep.k8s.io/4006
+	// alpha: v1.28
+	//
+	// Enables StreamTranslator proxy to handle WebSockets RemoteCommand/V5
+	// upgrade requests.
+	ClientRemoteCommandWebsockets featuregate.Feature = "ClientRemoteCommandWebsockets"
```

Comment:
since this isn't a generic apiserver feature, this should move to kube_features.go now that the checks of it are in pkg/registry/core/pod/rest/subresources.go

Reply:
Just to be clear, you're talking about `pkg/features/kube_features.go`  instead of this currrent `staging/src/k8s.io/apiserver/pkg/features/kube_features.go` -- right?

Reply:
yes, this needs to move to `pkg/features/kube_features.go`